### DISCUSSION TITLE
Is there any new driver version soon?

### DISCUSSION THREAD

**Discussion by Gr33n93** [Louisvdw/dbus-serialbattery#531] [Q&A] [answered]
My English is not so good. Hence Google translator.

Hello,

1. I would like to know if the upcoming driver version saves the names of the battery packs?
2. Will a summary of the batteries be installed in the upcoming version of the driver?
3. Do you already know when a new driver version will come?

Thank you for the great work!

**Comment by mr-manuel** [comment]
English please

**Reply by Gr33n93** [reply]
I threw it into Google translator for the

**Reply by mr-manuel** [reply]
You can use deepl.com in this case. Works far better :)

**Reply by Gr33n93** [reply]
Ok, thank you very much.



Can you still answer thens? questio

**Comment by mr-manuel** [comment, marked as answer]
1. Probably there will be some sort of workaround connected to the port. If the serial adapter changes USB port then the name is lost.
2. No, there exists a few workarounds in the meanwhile. Check [Dr-Gigavolt/dbus-aggregate-batteries](https://github.com/Dr-Gigavolt/dbus-aggregate-batteries), [mr-manuel/venus-os_dbus-mqtt-battery](https://github.com/mr-manuel/venus-os_dbus-mqtt-battery)
3. No, since Louisvdw manages this. Currently the most up-to-date version is in the jkbms_ble branch. You can also install it, if you are not using Bluetooth or JKBMS. See https://github.com/Louisvdw/dbus-serialbattery/discussions/485.

**Reply by mr-manuel** [reply]
A new version is released
